### PR TITLE
Comments for Backend 

### PR DESCRIPTION
**Pull request by simranmalik-4**

### CODE REVIEW

File: src/main/java/controller/GetMessage.java [hunk 1]
```diff
@@ -62,27 +86,25 @@ public void getMessageWithoutUserIdPathVariable(HttpServletRequest request) {
...
             throw new UserIdDoesNotExistException(path);
         }
         
-        //check if the passed chatId is valid
         if (!queryChat.checkIfChatIdExists(chatId)) {
             throw new ChatIdDoesNotExistException(path);
         }
 
-        //check if user is part of chat
+        /*
```
[the remark this answers is not in the document]
Comment (AgamAgarwal) [reply src/main/java/controller/GetMessage.java:97]: That's not true.

`// ...` and `/* ... */` comments are implementation comments while `/** ... */` comments are javadoc comments (used for describing what a function or a class does, or what a class member is for).

File: src/main/java/main/MainClass.java [hunk 2]
```diff
@@ -3,13 +3,12 @@
...
-//this annotation tells that this class is the starting point of our application 
-//the argument passed are the packages which we would like Spring to scan for Spring components. 
-//Refer: https://smarterco.de/java-spring-boot-mvc-ontroller-not-called/
 @SpringBootApplication(scanBasePackages = {"entity", "controller", "dbaccessor", "exceptionhandler", "helper", "main"})
 public class MainClass {
     public static void main(String[] args) {
-        //to start running the application
+        /*
+         * Starts running the application.
+        */
```
Comment (AgamAgarwal) [inline src/main/java/main/MainClass.java:11]: FYI: As @sankalpbansal-google mentioned, you can continue to use `// ...` style for such small implementation comments.

Comment (simranmalik-4) [reply src/main/java/main/MainClass.java]: Done

File: src/main/java/controller/GetMessage.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (simranmalik-4) [reply src/main/java/controller/GetMessage.java]: Changed syntax of short single line comments to //

**Comment by AgamAgarwal** [comment]
@simranmalik-4 This file has some conflicts with the master branch in the file `src/main/java/dbaccessor/user/UserAccessor.java` which will prevent the merge.

You may wanna merge master branch into this branch and resolve the conflicts.

File: src/main/java/controller/GetMessage.java [hunk 3]
```diff
@@ -21,6 +21,18 @@
...
 
+/**
+ * Controller which responds to client requests to get the details of a message.
+ * The response contains:
+ * (1)  MessageId
+ * (2)  ChatId
+ * (3)  SentByCurrentUser
+ * (4)  ContentType
+ * (5)  TextContent
+ * (6)  SentTs
```
[the remark this answers is not in the document]
Comment (simranmalik-4) [reply src/main/java/controller/GetMessage.java]: Done

File: src/main/java/controller/GetChat.java [hunk 4]
```diff
@@ -17,6 +17,13 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.PathVariable;
 
+/**
+ * Controller which responds to client requests to get the details of a chat.
+ * The response contains:
+ * (1)  ChatId
+ * (2)  LastSentMessageId
+ * (3)  CreationTs
```
[the remark this answers is not in the document]
Comment (simranmalik-4) [reply src/main/java/controller/GetChat.java]: Done

**Pull request merged**